.github/workflows/conformance-aks-v1.10.yaml

```diff
@@ -79,20 +79,23 @@ jobs:
     outputs:
       tested: ${{ steps.tested-tree.outputs.src }}
     steps:
-      - name: Retrieve pull request's base and head
-        if: ${{ github.event.issue.pull_request }}
-        id: pr
-        run: |
-          curl ${{ github.event.issue.pull_request.url }} > pr.json
-          echo "::set-output name=base::$(jq -r '.base.sha' pr.json)"
-          echo "::set-output name=head::$(jq -r '.head.sha' pr.json)"
       # Because we run on issue comments, we need to checkout the code for
       # paths-filter to work.
       - name: Checkout code
         if: ${{ github.event.issue.pull_request }}
         uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
         with:
           persist-credentials: false
```

Comment:
Huh, I'm just noticing this now, but how does this even work? If no SHA is passed to `actions/checkout`, it should checkout `GITHUB_REF` right? Which is the tip of the `master` branch for `issue_comment`-triggered workflows 🤔

Reply:
Yep, that's fine. We just need the source with `.git/` to be there to run `git rev-parse` and then for `dorny/paths-filter` to do its magic.

Reply:
Ah yes, you're right.